index.bs

```diff
@@ -11579,7 +11583,8 @@ if the interface has a [=setlike declaration=].
 
 <h5 id="es-forEach">forEach</h5>
 
-If the [=interface=] has any of the following:
+If the [=interface=] does not declare a [=regular operation=] with [=identifier=]
+<code class="idl">forEach</code> and has any of the following:
```

Comment:
This change is not needed, since things named "forEach" are disallowed in that case.  Similar for the other methods...